server_admin/topics/clients/client-policies.adoc

```diff
@@ -15,28 +15,28 @@ Client Policies realize the following points mentioned as follows.
-    registration and configuration ways. In the future, Client Registration will be replaced by Client Policies in the future.
+    registration and configuration ways. The current plans are for Client Registration to be replace by Client Policies.
 
 Validation of client configurations::
-    The {project_name} supports validation whether the client follows settings like Proof Key for Code Exchange,
-    Request Object Signing Algorithm, Holder-of-Key Token, etc on some endpoints like Authorization Endpoint, Token Endpoint, etc.
-    These can be specified by each setting item (on Admin Console, switch, pulldown menu, etc). To make the client application secure, administrator needs to set
-    lot of such settings in appropriate way, which makes it difficult for the administrator to secure their client application.
+    {project_name} supports validation whether the client follows settings like Proof Key for Code Exchange,
+    Request Object Signing Algorithm, Holder-of-Key Token, and so on on some endpoints like Authorization Endpoint, Token Endpoint, and so on.
+    These can be specified by each setting item (on Admin Console, switch, pulldown menu and so on). To make the client application secure, the administrator needs to set
+    many settings in the appropriate way, which makes it difficult for the administrator to secure the client application.
     Client Policies can do these validation of client configurations mentioned just above and they can also be used to auto-configure some client configuration switches to meet
     the advanced security requirements. In the future, individual client configuration settings may be replaced by Client Policies directly performing required validations.
 
 Conformance to a required security standards and profiles such as FAPI::
     The _Global client profiles_ are client profiles pre-configured in {project_name} by default. They are pre-configured to be compliant with standard security profiles like link:{adapterguide_link}#_fapi-support[FAPI],
     which makes it easy for the administrator to secure their client application to be compliant with the particular security profile. At this moment, {project_name} has global
-    profiles for the support of FAPI 1 specification. Administrator will just need to configure the client policies to specify which clients should
-    be compliant with the FAPI. Administrator can configure his own client profiles and client policies, so that {project_name} clients can be easily made compliant with various other
-    security profiles like SPA, Native App, Open Banking etc.
+    profiles for the support of FAPI 1 specification. The administrator will just need to configure the client policies to specify which clients should
+    be compliant with the FAPI. The administrators can configure their own client profiles and client policies, so that {project_name} clients can be easily made compliant with various other
```

Comment:
I'm not sure we should use "administrators" (plural), or at least not mix it with singular. We now sometimes use "administrator" and sometimes "administrators". I think we can use singular together with "they" as a common "workaround" if we don't want to specify he/she. @andymunro Could you please confirm this?

Reply:
Changing to singular and removing "his own" as @andymunro proposed above https://github.com/keycloak/keycloak-documentation/pull/1194#pullrequestreview-675651471